__tests__/integrations/Session/AuthenticationUser.test.ts

```diff
@@ -31,41 +30,46 @@ describe("Authentication User", () => {
   });
 
   it("Check user authentication with invalid email", async () => {
-    const response = await request.post("/v1/authenticate").send({
-      email: faker.internet.email(),
-      password,
-    });
-
-    expect(response.status).toBe(404);
-    expect(response.body.error).toBeTruthy();
-    expect(response.body.message).toBe("User not found");
-  });
-
-  it("Check user authentication without sending the email field", async () => {
-    const response = await request.post("/v1/authenticate").send({ password });
+    const InvalidEmail = faker.internet.email();
```

Comment:
```suggestion
    const email = faker.internet.email();
```

Variables must start with lowercase letters.

> remember to change that on line 36